blog/articles/amazon-ses-ec2-rails.md

```diff
@@ -0,0 +1,114 @@
+# Deliver Email with Amazon SES on EC2 for Rails
+
+[Amazon Simple Email Service][ses] (SES) is a
+high availability transactional email service.
+
+[ses]: https://aws.amazon.com/ses/
+
+When sending email from an EC2 instance,
+it is [particularly inexpensive][pricing]:
+
+* $0 for the first 62,000 emails sent per month
+* $0.10 per 1,000 emails sent thereafter
+
+[pricing]: https://aws.amazon.com/ses/pricing/
+
+It is available in three AWS regions:
+
+* eu-west-1 (Ireland)
+* us-east-1 (Northern Virginia)
+* us-west-2 (Oregon)
+
+If you have an AWS account,
+go to [SES in your AWS console][home].
+
+[home]: https://console.aws.amazon.com/ses/home
+
+## Move out of sandbox mode
+
+By default, SES in each region is in sandbox mode.
+Since sandbox mode only allows delivering to
+a whitelist of verified email addresses,
+it's unusable for a production web application.
+
+Go to the "Sending Statistics" page.
+click "Request a Sending Limit Increase"
+to open a support request.
+It may take hours, up to a day or two, for the request to be approved.
```

Comment:
Thoughts on changing this up a bit. The `up to a day or two` seems like an extension, or explanation to the previous statement, but they don't necessarily go together.

Maybe something like this:

> It may take hours or, in some cases, up to a day or two for the request to be approved.